### ISSUE TITLE
project collections from git fail to install

### ISSUE THREAD

**Issue by haught** [ansible/awx#9075]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
When using a project from git that has collections from git the syncing fails

##### ENVIRONMENT
* AWX version: 16.0.0
* AWX install method: docker
* Ansible version:  2.9.16, 2.10.4 custom_env
* Operating System: Linux
* Web Browser: Chrome

##### STEPS TO REPRODUCE

Have a git project containing
collections/requirements.yml
```yaml
---
collections:
  - name: https://github.com/ansible-collections/community.network.git
    type: git
    version: main
```
manually installing works fine: ```ansible-galaxy collection install -r collections/requirements.yml -p collections```

syncing project fails with error below

##### EXPECTED RESULTS

AWX will pull in collections from git

##### ACTUAL RESULTS

```
ERROR! Collection artifact at &apos;/var/lib/awx/.ansible/tmp/ansible-local-1358p3f_xw4a/tmp8w7_acap/community.network9yd8511k.git&apos; is not a valid tar file.
```

##### ADDITIONAL INFORMATION

The reason I am doing this is to pull in my own internal github repo that also fails in the same way but that can't be reproduced by you. Pulling in ```- name: cisco.ios``` and others works fine - just the git ones in a collection are failing. Roles from git work fine.

**Comment by wenottingham** [comment]
This isn't a supported feature of ansible 2.9, and we only ever use the base ansible venv for project updates. See https://github.com/ansible/awx/issues/8572#issuecomment-725664459

**Comment by alvaro-gh** [comment]
So, there's a requirements feature for collections which seems to be pretty straightforward and there's support for collection in AWX. I ran into the same issue and started following the issue thread which goes like this:
- This issue points to [#8572](https://github.com/ansible/awx/issues/8572).
- `#8572` points to [#8245](https://github.com/ansible/awx/issues/8245).
- `#8245` points to [#8598.](https://github.com/ansible/awx/pull/8598).

(Avoiding some links to keep text readable).

But I couldn't find anything pointing to a solution. Are we supposed to wait until Ansible 2.11 is used in AWX?
And I ran into this because of a simple bug in the Google Cloud collection which took a year to get fixed, collection which latest release goes back to February 2020.

So, end of rant. I suppose we have to handle this _edge_ case doing some manual work.
